pkg/quarterdeck/quarterdeck.go

```diff
@@ -203,6 +204,12 @@ func (s *Server) Routes(router *gin.Engine) (err error) {
 		return err
 	}
 
+	// Initialize prometheus collectors (this function has a sync.Once so it's safe to call more than once)
+	metrics.Setup()
```

Comment:
This is guaranteed to be called before any of the routes get called, with the possible exception of testing, but I think we do it in testing also for middleware verification.